js/lib/ui.widgets.js

```diff
@@ -1,223 +1,35 @@
-elgg.provide('elgg.ui.widgets');
+require(['elgg', 'jquery', 'elgg/widgets'], function (elgg, $, widgets) {
```

Comment:
This works as long as we inline elgg/widgets and this async function runs immediately. That's probably safe for BC and the cleanup is definitely easier later.